src/components/src/modals/load-data-modal.tsx

```diff
@@ -81,28 +106,24 @@ export function LoadDataModalFactory(
         {isCloudMapLoading ? (
           <LoadingDialog size={64} />
         ) : (
-          ElementType && <ElementType key={currentMethod?.id} intl={intl} {...props} />
+          ElementType && <ElementType key={currentMethod?.id} intl={intl} {...currentModalProps} />
         )}
       </StyledLoadDataModal>
     );
   };
 
-  LoadDataModal.defaultProps = {
-    onFileUpload: noop,
-    fileLoading: false,
-    loadingMethods: [
-      {
-        id: LOADING_METHODS.upload,
-        label: 'modal.loadData.upload',
-        elementType: FileUpload
-      },
-      {
-        id: LOADING_METHODS.storage,
-        label: 'modal.loadData.storage',
-        elementType: LoadStorageMap
-      }
-    ]
-  };
+  LoadDataModal.defaultLoadingMethods = [
```

Comment:
We still need this so that the custom LoadDataModal can access this via `LoadDataModal.defaultLoadingMethods`

https://github.com/keplergl/kepler.gl/pull/2679/files#r1794197038

But we don't need ti duplicated declaration in 2 places